scripts/scil_validate_bids.py

```diff
@@ -408,60 +404,44 @@ def main():
 
     data = []
     layout = BIDSLayout(args.in_bids, validate=False,
-                        ignore=_load_bidsignore_(args.in_bids))
+                        ignore=_load_bidsignore_(args.in_bids,
```

Comment:
Can you also address the upcoming deprecation warning on the ignore mechanism ? If using the BidsIndexer to load our ignores is possible that is.

Reply:
Also, make `args.in_bids` an aboslute path with `os.path.abspath(args.in_bids)`